src/quantity.rs

```diff
@@ -284,6 +284,49 @@ macro_rules! quantity {
             $description
         }
 
+        /// Unit enum.
+        #[allow(non_camel_case_types)]
+        #[non_exhaustive]
+        #[derive(Debug, Clone, Copy)]
+        pub enum UnitEnum {
```

Comment:
```suggestion
        pub enum Units {
```

Seems redundant to include `Enum` in the name. Does `Units` work without colliding with any other types?